packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -86,16 +87,27 @@ export class AtomicCategoryFacet
    * The number of values to request for this facet. Also determines the number of additional values to request each time this facet is expanded, and the number of values to display when this facet is collapsed.
    */
   @Prop() public numberOfValues = 5;
+  /**
+   * Whether this facet should contain a search box.
+   */
+  @Prop() public enableFacetSearch = false;
```

Comment:
I think we should make the default `true` because we want to show-case the feature. It should be opt-out rather than opt-in.